File: daemon/logdaemon.py
import time

count = 0
# Write a new message to the log/messages file every minute
while True:
    with open("/var/log/messages", "a") as logfile:
        logfile.write(f"A new log message from {__file__}: {count}\n")

    count += 1
    time.sleep(60)

File: daemon/logdaemon.c
#include <unistd.h>
#include <stdio.h>

FILE *f;

int main(void) {
    int count = 0;
    while(1) {
        f = fopen("/var/log/messages", "a");
        fprintf(f, "Log message written by logdaemon: Message %d\n", count);        
        fclose(f);

        sleep(30);
        count = count+1;
    }
}
